index.js

```diff
@@ -193,12 +189,13 @@ export default class Carousel extends Component {
     }
   }
 
-  _scrollTo = (offset, animated) => {
+  _scrollTo = (offset, animated, options) => {
+    const nofix = options && (options.nofix || false);
     if (this.scrollView) {
       this.scrollView.scrollTo({ y: 0, x: offset, animated });
 
       // Fix bug #50
-      if (Platform.OS === 'android' && !animated) {
+      if (!nofix && Platform.OS === 'android' && !animated) {
```

Comment:
The `nofix` option means to not execute the second `scrollTo` in `_scrollTo` used to fix bug #50. It is only used for the edge cases when switching to/from one of the Doppelgänger pages and we need to orchestrate the direction of the animation via a double `_scrollTo` elsewhere.